### PR TITLE
Complete overhaul of lavaland ruins.

### PR DESCRIPTION
**Pull request by AstraFarera**
## About The Pull Request
 Literally fills whole lavaland with buried in ashes necropolis deco with monsters, unpleasant stuff and useless loot.
 Old ruins remapped to themed ones.

 I've built about 120 ruin segments, but it's quite difficult for dumb worldgen to find enough place for all of them, so i removed way big and most useless of them, so there are just 104 left. Now most of them can spawn together at once.
 
## Why It's Good For The Game
 It's quite immersive. Good deco serves as nyoice rp background and freakin labirynth of ruins is nyoice for exploration.
 Also lots of hidden stuff that require bombs, or gibtonite ussage.
 Ruins filled with hi-chance-to-spawn-ore rocks, so it shall bring normal mining once again, while ore vents are now differs in 3 variants of each 3 boulder size types, as difficulty levels, placed in unhandy locations and require at least some pre-work.

## Proof of testing
image

## Changelog

:cl:
add: Added hella amount of lavaland ruins.
del: Removed old ruins from spawnlist.
/:cl:

### CODE REVIEW

**Comment by AstraFarera** [comment]
Well it's not about an icebox, so why unit test with it...
My job s done here, now waitin for checkin, approval and then lots of nerf/fix/removal work X)

**Comment by Iajret** [comment]
@AstraFarera слегка накуралесил модульностью, можешь глянуть как я разнёс файлы (я правда удаления из ТГшных не подписал комментариями, так как делал на пофиг). Есесно я ничего не тестировал и не проверял, спавнятся ли твои и убраны ли из спавна ТГшные руинки. Чуть позже, как разленюсь, уже проверю локально, чтобы убедиться - спавнятся ли эшволкеры и всё такое прочае.

**Comment by AstraFarera** [comment]
Seems there's no need n those.. Aaand Ijret totally ignores my dms.. Aaaand only thing i can play n this game is just mapping so.. whatever. n'ai rien à faire

**Comment by Kocma-san** [comment]
Хотелось бы и посмотреть на новые руины! Скринчики будутб? :3

**Comment by AstraFarera** [comment]
> Хотелось бы и посмотреть на новые руины! Скринчики будутб? :3

Ts test merged for a while ingame.. 

currently just noting vi bugs to fix.

**Comment by AstraFarera** [comment]
So much thingys to change..

**Comment by AstraFarera** [comment]
image

**Comment by AstraFarera** [comment]
Lack of fire enemies. Borgs still can go careless even with all those runes.
Demons seems nerfed waaay much with that ai switch.. aaand tey shall not use emotes. 
Random book thingy now way easy to get cause of that btw.. 
Frozen moonlight requires.. way significant gravity effect.. maybe 90% or so.. 
Dragon armour questionable.

uhum..

**Comment by AstraFarera** [comment]
Aight.. noted global dificulty n ways ppl move, learned enemy resistances and ai abuses. lt's see n next iteration journey.

**Comment by AstraFarera** [comment]
Finally..

**Comment by AstraFarera** [comment]
Yeaaap. Now its nyoice.

File: tgstation.dme [hunk 1]
```diff
@@ -8461,6 +8461,7 @@
 #include "tff_modular\modules\holidays_decor\halloween\halloween-decor.dm"
 #include "tff_modular\modules\holidays_decor\new_year\decorations.dm"
 #include "tff_modular\modules\interdyne-remap\interdyne.dm"
+#include "tff_modular\modules\lavaland_ruins\lavaland.dm"
```
Comment (Iajret) [inline tgstation.dme:8464]:
```suggestion
#include "tff_modular\modules\lavaland_ruins\items.dm"
#include "tff_modular\modules\lavaland_ruins\lavaland.dm"
#include "tff_modular\modules\lavaland_ruins\mobs.dm"
```

**Comment by AstraFarera** [comment]
pls remove all of my things from vi server entierly, i do not wanna create any for this kind of ppl

**Comment by robot-seriy** [comment]
Forgive our community, filled with rabble that cannot control their own emotions. Thank you for those days when the Necropolis became great again. E'Luna.

**Comment by MintRainLanayaRin** [comment]
It was fun to play in the beautiful lavaland. Beautiful dungeons... Big and cute lobsters-popstar. Glowing sword... blep~

Было много интересных штук, ради которых хотелось играть...
Спасибо тебе за множество интересных штук, жалко что поиграть с ними теперь получится только на локалке :<

**Comment by AstraFarera** [comment]
image

**Pull request closed without merging**